website/pages/docs/concepts/policies.mdx

```diff
@@ -393,6 +393,27 @@ options are:
     }
     ```
 
+  - It's important to note that the use of globbing can result in surprising
```

Comment:
Unsure if we should be using `can` or `may` ..

Reply:
`may` is much better. I'll change this.